src/PowerShellEditorServices/Services/Analysis/AnalysisService.cs

```diff
@@ -941,43 +438,68 @@ private static DiagnosticSeverity MapDiagnosticSeverity(ScriptFileMarkerLevel ma
         {
             switch (markerLevel)
             {
-                case ScriptFileMarkerLevel.Error:
-                    return DiagnosticSeverity.Error;
+                case ScriptFileMarkerLevel.Error:       return DiagnosticSeverity.Error;
+                case ScriptFileMarkerLevel.Warning:     return DiagnosticSeverity.Warning;
+                case ScriptFileMarkerLevel.Information: return DiagnosticSeverity.Information;
+                default:                                return DiagnosticSeverity.Error;
+            };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4961943)